source/apps.txt

```diff
@@ -20,6 +21,7 @@ Develop & Deploy Apps
    Deployment Models & Regions </apps/deployment-models-and-regions>
 
 - :doc:`Create an App </apps/create>`
+- :doc:`Get App Metadata </apps/metadata>`
 - :doc:`Update an App </apps/update>`
 - :doc:`Copy an App </apps/copy>`
 - :doc:`Delete an App </apps/delete>`
```

Comment:
nit (and feel free to not implement in this PR): can you make these refs instead of :doc: directives?

Reply:
On these pages I prefer the `:doc:` directive because the stuff between the backticks is a 1:1 copy paste of the toctree directive contents. Since we want this list to mirror the toctree I think it's prudent to keep these for minimal friction.